### PR TITLE
Msvc build

### PR DESCRIPTION
**Pull request by elpaso**
Builds on windows with QtCreator and MS C compiler (but crashes immediately)

### CODE REVIEW

File: src/qml/qgismobileapp.qml [hunk 1]
```diff
@@ -1342,6 +1342,7 @@ ApplicationWindow {
       }
     }
     onDropped: {
+      console.warn('Drop URL: ' + drop.urls[0])
```
Comment (elpaso) [inline src/qml/qgismobileapp.qml]: This can be removed....

**Comment by m-kuhn** [comment]
Nice, thanks!
Does this supersede #732?

**Comment by elpaso** [comment]
Yes, it adds the MSVC build changes on top of MXE build changes.

**Comment by m-kuhn** [comment]
Superseded by #793

**Pull request closed without merging**